lldb/include/lldb/DataFormatters/FormatClasses.h

```diff
@@ -96,6 +97,8 @@ class FormattersMatchCandidate {
 
   bool DidStripTypedef() const { return m_flags.stripped_typedef; }
 
+  uint32_t GetPtrStrippedDepth() const { return m_ptr_stripped_depth; }
```

Comment:
I think this reads better as GetPtrStrippingDepth than GetPtrStrippedDepth.  You aren't asking about something that has been done, but about a thing you are planning to do, so active seems more appropriate.